xeno/shield_removal.md

```diff
@@ -0,0 +1,94 @@
+
+Other buffs are more interesting. While shields may be strong, they are boring - all they do is just add extra health and let the xenos stick around on the front for longer. Other mechanics, such as speed boosts, clarity stacks (or even tgmc style increased evasion chance) would be more engaging, unique, and some would allow for more skillful use with them (i.e. a shield boost or a proc’ed next slash with a time expiry, like sentinel.) At the end of the day shields are lazy design that would be more interesting if replaced with a wider range of abilities.
+
+## Reasons against
+
+They allow xenos to have more frontline durability without being overpowered in one on one engagements. Okay, this primarily only applies to warriors, but it’s a good argument - buffing their health at all times could lead to them being OP in single engagements but also having little frontline power. (however the shield also still applies in one on ones)
+
+Makes game easier for newer players
+
+Xenos would be too squishy without them
+
+It’s the only thing that makes shaman good 
+
+## Goals
+
+1. To severely limit the number of castes that can possibly have shields, and make them all have clear telegraphs. This is accomplished by only leaving it on rav and crusher, which have said mechanics.
+
+2. To remove the ability of all other castes to get shields in normal gameplay. Accomplished by removing from warden and carrier
+
+3. To improve the game experience via making the reasons for shield removal irrelevant (by removing shields)
+
+## Non-goals
+
+1. Majorly affect side balance. I don't want to make a side stronger or weaker with this, which is why every caste will get compensatory buffs.
+
+2. Make warrior weak while frontlining. See above.
+
+## Content
+
+Shields to be removed from all but crusher, rav
+
+Warden shield - replace with a speed boost / warrior clarity stacks / fire extinguish? -> a buff that helps xenos at the front line, and can help them while they are at full health
+
+Carrier shield - replace with aoe clarity stacks / a 10s aoe clarity effect / slash damage -> some other kind of buff that makes xenos better offensively
+
+Vanguard can prob keep shield since it’s just a one free hit mechanic 
+
+Warrior should prob get like 5 extra armour and not get the health nerf from !2501
+
+Base Prae shield from spit also needs to be removed, it’s not super significant so maybe just a tiny health buff, but can prob be just fine with no change
+
+Shield fruit to be replaced with another type of short-term buff fruit (such as speed, damage, clarity, plasma, heal over time)
```

Comment:
you should be explaining what shields are going to do in your new overhaul.
Example: Shields will be a short lasting buff to reduce damage for a short period of time. You don't need to go into what's going to be changed about each caste because any change is acceptable as long as they align with your goals and don't step over any boundaries in your non-goals.

Reply:
content made more general, specific changes moved down into their own section